pkgs/build-support/fetchnextcloudapp/default.nix

```diff
@@ -1,10 +1,20 @@
-{ stdenv, fetchzip, applyPatches, ... }:
+{ stdenv, fetchzip, applyPatches, lib, ... }:
 { url
 , sha256
+, licenses
 , patches ? [ ]
 , name ? null
 , version ? null
+, description ? null
+, homepage ? null
 }:
+let
+  # TODO: do something better
+  licenseMap = {
```

Comment:
@alyssais could I have your opinion on approximating licenses this way? Is this bad or is there a way to make it OK?

Reply:
I don't suppose the upstream metadata could help clear this up?

Reply:
I see https://github.com/nextcloud/bookmarks/blob/master/COPYING so I think this is a nc4nix improvement maybe? It should get SPDX IDs as much as possible.

Reply:
https://github.com/nextcloud/bookmarks/blob/master/package.json#L50

Reply:
cc @armijnhemel